guide/private-key-management/multi-user-schemes.md

```diff
@@ -52,38 +52,41 @@ Just like with personal schemes, external signing devices can be used to hold on
 ### How it works
 A wallet or coordination application initiates a multi-sig wallet, choosing the number of total keys, and the number required to sign transactions. It then needs to add private keys from the other user's wallet. It can then complete the creation process. For any future transaction from the multi-sig wallet the required amount of co-signers need to sign (using Partially Signed Bitcoin Transactions - PSBT from [BIP174]({{ 'https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki'}}){:target="_blank"}) before any transaction is valid.
 
-#### Pros
+{% include fact/pros.html %}
+
 - Allows several people to access and control a bitcoin wallet
 - Can tailor requirements for multiple co-signers and access
 - High security, if setup well
 
-#### Cons
+{% include fact/close.html %}
+
+{% include fact/cons.html %}
+
 - Has significant complexity and op-sec burden for multiple private keys, each of which might need a good manual backup scheme
 
+{% include fact/close.html %}
+
 ### Best practice
 
-**When to use**
+#### When to use
```

Comment:
The fact component was designed for specifically this section ([see here](https://www.figma.com/file/K5ahSqVzoMVBuhT5hAVRJr/bitcoin.design?node-id=850%3A6300)), so I'd also switch "When to use", "When not to use", "Variations", and "Do’s" to the fact component. Creates a consistent appearance for this loose groups of bullet categories.

Reply:
I'll re-do this page applying the proper include, missed that